test/e2e/certs.go

```diff
@@ -32,6 +32,9 @@ var _ = Describe("MasterCerts", func() {
 	})
 
 	It("should have all expected certs on the master", func() {
+		if testContext.Provider == "" {
```

Comment:
Since this hasn't merged yet, I went ahead and created a shared util function in this PR. PTAL.